pkgs/applications/networking/browsers/chromium/darwin/default.nix

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Maybe we should add a chromium-bin instead, mirroring what we have for Firefox?

Reply:
For the uninitiated, what do these suffixes mean?

 - `-bin`
 - `-bin-unwrapped`
 - `-unwrapped`

Reply:
Nvm, found out https://nixos.wiki/wiki/Package_naming_conventions